### PR TITLE
media-libs/xvid: do not install static libraries

### PR DESCRIPTION
**Pull request by Alessandro-Barbieri**
Closes: https://bugs.gentoo.org/378311
Package-Manager: Portage-2.3.76, Repoman-2.3.17
Signed-off-by: Alessandro Barbieri <lssndrbarbieri@gmail.com>

### CODE REVIEW

File: media-libs/xvid/xvid-1.3.5.ebuild [hunk 1]
```diff
@@ -40,6 +40,9 @@ src_prepare() {
 		-e 's/@$(RM)/$(RM)/' \
 		-e 's/@$(INSTALL)/$(INSTALL)/' \
 		-e 's/@cd/cd/' \
+		-e '/\$(libdir)\/\$(STATIC_LIB)/d' \
+		-e 's/\$(BUILD_DIR)\/\$(STATIC_LIB)//g' \
+		-e 's/info \$(STATIC_LIB)/info/g' \
```
Comment (SoapGentoo) [inline media-libs/xvid/xvid-1.3.5.ebuild]: together with the EAPI bump, this step is rather invasive. Just revbump, drop to unstable and then I can commit it.

Comment (Alessandro-Barbieri) [reply media-libs/xvid/xvid-1.3.5.ebuild]: done

**Comment by Alessandro-Barbieri** [comment]
ping

**Comment by lu-zero** [comment]
It would need a [static-libs](https://packages.gentoo.org/useflags/static-libs) useflag.

**Comment by Alessandro-Barbieri** [comment]
@lu-zero it has no reverse dependencies needing it so it should not be used
see https://projects.gentoo.org/qa/policy-guide/installed-files.html?highlight=static#pg0302
@SoapGentoo

**Pull request closed without merging**